### PR TITLE
Persian transliteration

### PR DESCRIPTION
**Pull request by haditim**
I have reverted back Persian transliteration to remove the conflict with Arabic. However, some missing characters are now replaced. Hope it solves the test issue now.

### CODE REVIEW

**Comment by cpfeiffer** [comment]
Does this PR supersede #1246? If yes, you can close it. Next time, you can simply add more commits to the same branch or force push. This will update the existing PR, so no need to open a new one.

**Comment by haditim** [comment]
Yes this should supersede #1246. However, the language detection which is requested there would be highly appreciated. Or another checkbox in the settings to transliterate Persian instead of Arabic. That is why I did not close that one. Although I myself cannot proceed further with that.

File: app/src/main/java/nodomain/freeyourgadget/gadgetbridge/util/LanguageUtils.java [hunk 1]
```diff
@@ -76,9 +76,8 @@
             put('و', "w"); put('ي', "y"); put('ى', "a"); put('ﺓ', "");
             put('آ', "2"); put('ئ', "2"); put('إ', "2"); put('ؤ', "2"); put('أ', "2"); put('ء', "2");
 
-            // Farsi
-            put('پ', "p"); put('چ', "ch"); put('ڜ', "ch"); put('ڤ', "v"); put('ڥ', "v");
```
[the remark this answers is not in the document]
Comment (haditim) [reply app/src/main/java/nodomain/freeyourgadget/gadgetbridge/util/LanguageUtils.java]: I do not know where they belong to, but definitely not Persian :D.

File: app/src/main/java/nodomain/freeyourgadget/gadgetbridge/util/LanguageUtils.java [hunk 2]
```diff
@@ -76,9 +76,8 @@
...
-            put('ڨ', "g"); put('گ', "g"); put('ݣ', "g");
+            // Persian(Farsi)
+            put('پ', "p"); put('چ', "ch"); put('ژ', "zh"); put('گ', "g"); put('ی', "y"); put('؟', "?");
```
[the remark this answers is not in the document]
Comment (haditim) [reply app/src/main/java/nodomain/freeyourgadget/gadgetbridge/util/LanguageUtils.java]: Actually my ی (= 06CC) is different than yours (0649). That's why we did not have any assertion error (you can put both in a word and check the difference e.g. یار vs. ىار). You can keep yours as it is although I could argue that even in Arabic ربی should be transliterated to Rabbi or Rabbi and not Rabba.

[the remark this answers is not in the document]
Comment (haditim) [reply app/src/main/java/nodomain/freeyourgadget/gadgetbridge/util/LanguageUtils.java]: Thanks for comment. It's done. I had also forgotten to add ک which is taken care of now.

**Comment by cpfeiffer** [comment]
Please guide me how to proceed with this, i.e. add a comment when this PR is ready to be merged and ideally create a new PR with those parts of #1246 that are needed.

Do you also notice #1233 that I just merged?

**Comment by haditim** [comment]
Hey,
This PR looks ready to be merged from my side. #1246 has another story and should either be ignored for now or an option in the app settings should be added that takes transliteration mappings from Persian by default as opposed to Arabic which is now the default and a bit non-understandable for Persians. But I cannot do this as my android development skills are at zero. 
I suggest you go ahead and merge this and #1233 for which I have a bit of difficulty understanding what it exactly does. To me RTL makes sense when there's no transliteration. Maybe I'm missing something?
Thanks

**Comment by cpfeiffer** [comment]
Yes, AFAIU this is for devices which actually support these character sets, where transliteration is not needed.

**Comment by cpfeiffer** [comment]
So, there we go, it's merged :-) Thanks a lot!

**Comment by cpfeiffer** [comment]
I hope I didn't mess things up with all these PRs getting merged. Please test, if you can :-)

**Comment by cpfeiffer** [comment]
I just noticed that the commit from #1246 was actually part of this PR as well.

**Pull request merged**